File: Database/Assembly_SRPG/MagnificationParam.cs
namespace SRPG
{
    using System;

    public class MagnificationParam
    {
        public string iname;
        public int[] atkMagnifications;

        public MagnificationParam()
        {
            base..ctor();
            return;
        }

        public void Deserialize(JSON_MagnificationParam json)
        {
            if (json != null)
            {
                goto Label_000C;
            }
            throw new InvalidJSONException();
        Label_000C:
            this.iname = json.iname;
            this.atkMagnifications = json.atk;
            return;
        }
    }
}

File: Database_V2/ParamFunctions_untested/MagnificationParam.py
def MagnificationParam(json):
    this={}#MagnificationParamjson)
    #if(json==null)
    #thrownewInvalidJSONException()
    if 'iname' in json:
        this['iname'] = json['iname']
    if 'atk' in json:
        this['atkMagnifications'] = json['atk']
return this
